File: deaf_grandma.py
from operator import truediv


def deaf_grandma():
    words = True
    goodbyes = 0

    while(words):
        user_input = input("Hey Kid!")

        if user_input == "":
            input("WHAT!?")
        elif user_input.upper() != user_input:
            input("SPEAK UP, KID!")
        elif user_input == "GOODBYE!":
            goodbyes += 1
            if(goodbyes == 1):
                input("LEAVING SO SOON?")
            elif(goodbyes == 2):
                input("LATER SKATER!")
                words == False
        elif user_input.upper() == user_input:
            input("NO, NOT SINCE 1946!")



deaf_grandma()

File: deaf_grandma.js
function deafGrandma() {
    let words = true
    let goodbyes = 0

    while (words) {
        
        let userInput = prompt("Hey Kid!")

        if(userInput === ""){
            alert("WHAT!?")
        } else if(userInput.toUpperCase() !== userInput){
            alert("SPEAK UP, KID!")
        } else if(userInput === "GOODBYE!"){
            goodbyes++
            if(goodbyes === 1){
                alert("LEAVING SO SOON?")
            } else if(goodbyes === 2){
                alert("LATER, SKATER!")
                words = false
            }
        } else if(userInput.toUpperCase() === userInput){
            alert("NO, NOT SINCE 1946!")
        }


    }
    document.writein("Program exited.")
}

deafGrandma();